src/server/internalServer.cpp

```diff
@@ -447,6 +447,11 @@ std::string noSuchBookErrorMsg(const std::string& bookName)
   return "No such book: " + bookName;
 }
 
+std::string noSearchResultsMsg(const std::string& pattern)
+{
+  return "There is no article with the title <b>\"" + encodeDiples(pattern) + "\"</b> and the fulltext search engine is not available for this content.";
```

Comment:
With #497, we don't redirect to article titled with `pattern`.
We should rewrite this message with "The fulltext search engine is not available for this content" only.

Reply:
Done